parcel.go

```diff
@@ -13,34 +13,67 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	newParcel, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		return 0, err
+	}
+
+	lastId, err := newParcel.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+
+	return int(lastId), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
-
-	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
 
+	getParcel := s.db.QueryRow("SELECT * FROM parcel WHERE number = :number",
+		sql.Named("number", number))
+	err := getParcel.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		return p, err
+	}
+
 	return p, nil
 }
 
 func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
-	// реализуйте чтение строк из таблицы parcel по заданному client
-	// здесь из таблицы может вернуться несколько строк
-
-	// заполните срез Parcel данными из таблицы
 	var res []Parcel
 
+	getParcels, err := s.db.Query("SELECT * FROM parcel WHERE client = :client",
+		sql.Named("client", client))
+	if err != nil {
+		return res, err
```

Comment:
Лучше при ошибке возвращать `nil`
```suggestion
		return nil, err
```

Reply:
исправлено